website/templates/dashboard.mako

```diff
@@ -1,69 +1,22 @@
 <%inherit file="base.mako"/>
 <%def name="title()">Dashboard</%def>
 
+<%def name="container_class()">container-xxl</%def>
```

Comment:
Yes, it allows certain pages on osf to go up to 1600px wide like the Wiki. This page needs it too.